src/pretix/plugins/sendmail/views.py

```diff
@@ -36,13 +41,41 @@ def form_valid(self, form):
         self.request.event.log_action('pretix.plugins.sendmail.sent', user=self.request.user, data=dict(
             form.cleaned_data))
 
+        tz = pytz.timezone(self.request.event.settings.timezone)
+
         failures = []
+        self.output = {}
         for o in orders:
-            try:
-                mail(o.email, form.cleaned_data['subject'], form.cleaned_data['message'],
-                     None, self.request.event, locale=o.locale, order=o)
-            except SendMailException:
-                failures.append(o.email)
+            if self.request.POST.get("action") == "preview":
+                for l in self.request.event.settings.locales:
+                    with language(l):
+                        self.output[l] = []
+                        self.output[l].append(
+                            form.cleaned_data['subject'].localize(l))
+                        message = form.cleaned_data['message'].localize(l)
+                        preview_text = message.format(
+                            order='ORDER1234',
+                            event=self.request.event.name,
+                            order_date=date_format(now(), 'SHORT_DATE_FORMAT'),
+                            due_date=date_format(now() + timedelta(days=7), 'SHORT_DATE_FORMAT'),
+                            order_url='link_to_order_url')
```

Comment:
I did and I thought I changed it...  Do you see [the new code in here](https://github.com/pretix/pretix/pull/279/commits/1261f00eb60ce5a4bc5543d3d72be0c2b5283c6b) or is it gone?